src/main/java/com/impactupgrade/integration/hubspot/model/ContactProperties.java

```diff
@@ -1,21 +1,43 @@
 package com.impactupgrade.integration.hubspot.model;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class ContactProperties implements AbstractModel {
 
   private HasValue<String> firstname;
   private HasValue<String> lastname;
+  @JsonProperty("email") 
```

Comment:
Super picky: If the JSON property name and the Java field name are identical, the @JsonProperty annotation isn't technically needed. So for `email`, `phone`, `address`, etc., remove them?

Reply:
(They're only needed if the JSON and Java names differ -- ie, camel vs. snake case)